plugins/calibre.koplugin/main.lua

```diff
@@ -123,6 +128,23 @@ function Calibre:addToMainMenu(menu_items)
     end
 end
 
+-- metadata options
+function Calibre:getMetadataMenuTable()
+    return {
+        {
+            text = _("Enable calibre to doc sidecar path map"),
+            checked_func = function()
+                return G_reader_settings:nilOrTrue("calibre_metadata_keep_sidecar_path_map")
+            end,
+            help_text = _(
+[[If you are using Calibre koreader plugin, this option is required to make it work properly if you're not using the default document sidecar settings.]]),
```

Comment:
The text as written seems to imply it should be automatically turned on or off depending on some other setting?

Reply:
Yea, if you connect wirelessly or do a Calibre search, you initialise CalibreMetadata, which in turn populates this file.

It might also be handy to add a menu item to explicitly create this file, to cover the use case of users who don't use wireless connection or use Calibre metadata search. 

The workflow would then be: 
1. Explicitly run the menu action to populate the file
2. Physically connect the device to Calibre